suites/reef/cephfs/tier-3_cephfs_bugs.yaml

```diff
@@ -18,6 +18,7 @@
   # CEPH-83583762: Check to prevent usage of Pools which has Pool level snaps for Filesystem creation.
   # CEPH-83584084: Validate if no repeated mclientscaps(revoke) entries found in MDS.
   # CEPH-83586730: validate standby replay node not removed after setting mds_inject_health_dummy
+  # CEPH-83583721 - Validate the details of all MDS servers are displayed with command `ceph mds metadata`.
```

Comment:
Can you include # CEPH-83583721 also in this ?

Reply:
updated the suite file